File: src/main/java/com/example/administrator/recyclerview/main/MainViewPagerAdapter.java
package com.example.administrator.recyclerview.main;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;

public class MainViewPagerAdapter extends FragmentStatePagerAdapter {

    public MainViewPagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        return MainFragment.newInstance(position);
    }

    @Override
    public int getCount() {
        return 4;
    }
}

File: RecyclerMain_Kotlin/kotlin/com/example/administrator/recyclerview/main/main/MainViewPagerAdapter.kt
package com.example.administrator.recyclerview.main.main

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter

class MainViewPagerAdapter(fm: FragmentManager) : FragmentStatePagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        return MainFragment.newInstance(position)
    }

    override fun getCount(): Int {
        return 4
    }
}
